### PR TITLE
chore: fixed misleading usage information

### LINKED ISSUE #2138: Misleading usage information
As of now `newman -h` results in the following:-

```sh
error: no arguments provided

Usage: newman [options] [command]

Options:
  -v, --version               output the version number
  -h, --help                  output usage information

Commands:
  run [options] <collection>  URL or path to a Postman Collection.

To get available options for a command:
  newman [command] -h
```

Usage information should be `newman <command> [options]` instead.

### PR DESCRIPTION
**Pull request by jamesgeorge007**
closes #2138

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
\cc @codenirvana

File: bin/newman.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const _ = require('lodash'),
 
 program
     .version(version, '-v, --version')
+    .usage('[command] [options]')
```
[the remark this answers is not in the document]
Comment (jamesgeorge007) [reply bin/newman.js]: without explicitly specifying it would be:-
```bash
Usage: newman [options] [command]
```

**Comment by jamesgeorge007** [comment]
\cc @codenirvana

File: bin/newman.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (codenirvana) [reply bin/newman.js]: Got it, but I don't see anything wrong there as both are optional and order here doesn't actually make any difference.

**Comment by jamesgeorge007** [comment]
\cc @codenirvana

**Comment by shamasis** [comment]
@jamesgeorge007 ... by the way, thanks for your contribution.

**Pull request merged**